components/CallToAction/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { StyledCallToAction } from './StyledCallToAction'
+import { useState } from 'react'
+import { SpeedDial, SpeedDialIcon, SpeedDialAction } from '@mui/material'
+import RateReviewIcon from '@mui/icons-material/RateReview'
+import MessageIcon from '@mui/icons-material/Message'
+import { setDrawer } from '@/lib/redux/slices/drawer'
+import { useDispatch, useSelector } from 'react-redux'
+import { DrawerComponentProps } from '@/utils/types/redux'
+interface SpeedDialProps {
+    icon: any
+    label: string
+    drawerDisplay: 'message' | 'testimonial' | null
+}
+
+const SpeedDialOptions: SpeedDialProps[] = [
+    { icon: <MessageIcon />, label: 'Contact Me', drawerDisplay: 'message' },
```

Comment:
this `drawerDisplay` should be `client-request`. We should create constants for these in the utils folder, `utils/constants`.

```
export const DRAWER_TYPES = {
  CLIENT_REQUEST: 'CLIENT_REQUEST',
  TESTIMONIAL: 'TESTIMONIAL'
}
```